ambuda/seed/texts/mahabharata.py

```diff
@@ -8,6 +8,25 @@
 
 BASE_URL = "https://bombay.indology.info/mahabharata/text/UD/MBh{n}.txt"
 
+TEI_HEADER = """<teiHeader xml:lang="en">
+  <fileDesc>
+    <titleStmt>
+        <title>Mahābhārata</title>
+        <author>Vyāsa</author>
+    </titleStmt>
+
+    <publicationStmt>
+        <publisher>bombay.indology.info</publisher>
+        <availability>
+        <p>The electronic text of the Mahābhārata is Copyright (C) The Bhandarkar
+        Oriental Research Institute (BORI), Pune and is sourced from <a href="http://bombay.indology.info/mahabharata/statement.html">John Smith's website.</a></p>
```

Comment:
`<a>` is not a TEI element, so this isn't a valid header. I think the TEI approach is to use `<ref target="...">` instead:

https://tei-c.org/release/doc/tei-p5-doc/en/html/SA.html#SAUR

I want these headers to be compatible with the spec to be consistent with the view that what we're showing is just a chopped-up version of a valid TEI document. So in addition to changing the `<a>` here, we'll also need to update `xml.py` to interpret `ref` and reshape it.